File: app/src/main/java/com/wf/mn/home/view/MyAdapter.java
package com.wf.mn.home.view;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.alibaba.android.vlayout.DelegateAdapter;
import com.alibaba.android.vlayout.LayoutHelper;
import com.wf.mn.R;

/**
 * @author : wf
 * @time : 2020-11-13-14:46
 */
public class MyAdapter extends DelegateAdapter.Adapter {

    private LayoutHelper helper;

    public MyAdapter(LayoutHelper helper) {
        this.helper = helper;
    }

    @Override
    public LayoutHelper onCreateLayoutHelper() {
        return helper;
    }

    @NonNull
    @Override
    public RecyclerView.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {

        View inflate = LayoutInflater.from(parent.getContext()).inflate(R.layout.search_layout, parent, false);

        return new MyViewHoler(inflate);
    }

    @Override
    public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int position) {




    }

    @Override
    public int getItemCount() {
        return 1;
    }


    class MyViewHoler extends RecyclerView.ViewHolder{

        public MyViewHoler(@NonNull View itemView) {
            super(itemView);
        }

    }

}

File: app/src/main/java/com/wf/mn/home/view/MyAdapter.kt
package com.wf.mn.home.view

import android.icu.number.NumberFormatter.with
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.wf.mn.R

/**
 *@time : 2020-11-21-10:06
 *@author : wf
 */

class MyAdapter : RecyclerView.Adapter<MyAdapter.MyHolder>() {
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MyAdapter.MyHolder {

        return MyHolder(LayoutInflater.from(parent.context).inflate(R.layout.activity_login,parent,false))
    }

    override fun getItemCount(): Int {
        return 13;
    }

    override fun onBindViewHolder(holder: MyAdapter.MyHolder, position: Int) {




    }

    class MyHolder(item:View) : RecyclerView.ViewHolder(item)


}
